### PR TITLE
docs: document how to set up Jenkins with Pipeline promotion events

### PR DESCRIPTION
**Pull request by yitsushi**
**What changed?**
An extra page added to the documentation website on how to set up pipeline notifications to trigger a jenkins job.

**Why was this change made?**
Document a feature.

**How was this change implemented?**
Created a new file

**How did you validate the change?**
Checked the page with `yarn start`

**Release notes**
NA

**Documentation Changes**

### CODE REVIEW

File: website/docs/pipelines/pipelines-with-jenkins.mdx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import TierLabel from "./../_components/TierLabel";
+
+import CodeBlock from "@theme/CodeBlock";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+
+# Setting Up Pipelines to Notify a Jenkins Webhook <TierLabel tiers="Enterprise" />
+
+With the [Notification
+Controller](https://fluxcd.io/flux/components/notification/), a Jenkins Webhook
+can be invoked on Pipepine promotion events.
```
Comment (yiannistri) [inline website/docs/pipelines/pipelines-with-jenkins.mdx:15]:
```suggestion
can be invoked on Pipeline promotion events.
```

File: website/docs/pipelines/pipelines-with-jenkins.mdx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+---
+
+import TierLabel from "./../_components/TierLabel";
+
+import CodeBlock from "@theme/CodeBlock";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+
+# Setting Up Pipelines to Notify a Jenkins Webhook <TierLabel tiers="Enterprise" />
+
+With the [Notification
```
Comment (yiannistri) [inline website/docs/pipelines/pipelines-with-jenkins.mdx:13]:
```suggestion
Using the [Notification
```

**Review by yiannistri — APPROVED** [review]

File: website/docs/pipelines/pipelines-with-jenkins.mdx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+---
+
+import TierLabel from "./../_components/TierLabel";
+
+import CodeBlock from "@theme/CodeBlock";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+
+# Setting Up Pipelines to Notify a Jenkins Webhook <TierLabel tiers="Enterprise" />
+
+Using the [Notification
```
Comment (Callisto13) [inline website/docs/pipelines/pipelines-with-jenkins.mdx:13]:
```suggestion
Using Flux's [Notification
```

File: website/docs/pipelines/pipelines-with-jenkins.mdx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  "reason": "Promote",
+  "reportingController": "pipeline-controller",
+  "reportingInstance": "chart-pipeline-controller-8549867565-7822g"
+}
+```
+
+## Configure Notification Provider
+
+In order to be able to invoke a generic webhook, a notification provider has to
+be defined. Jenkins expects the secret token as a GET parameter or in the
```
Comment (Callisto13) [inline website/docs/pipelines/pipelines-with-jenkins.mdx:63]:
```suggestion
be defined. Jenkins expects the secret token which you configured above as a GET parameter or in the
```

**Review by Callisto13 — APPROVED** [review]
LGTM

This is part of a series of "pipelines with x" yes?

**Comment by yitsushi** [comment]
> This is part of a series of "pipelines with x" yes?

It seems yes, but i don't know if we go for more than jenkins and tekton for a while.

**Comment by Callisto13** [comment]
👍 sounds good

**Pull request merged**